RetailCoder.VBE/UI/FindSymbol/FindSymbolDialog.cs

```diff
@@ -26,6 +26,8 @@ private void viewModel_Navigate(object sender, NavigateCodeEventArgs e)
         public FindSymbolDialog()
         {
             InitializeComponent();
+
+            this.Text = string.Format("Rubberduck - {0}", RubberduckUI.FindSymbolDialog_Caption);
```

Comment:
As this string has not yet be translated it is currently only available in German.
As soon as for the other languages the translation will be continued this will be filled there.